staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go

```diff
@@ -302,6 +302,7 @@ func (le *LeaderElector) release() bool {
 	}
 	leaderElectionRecord := rl.LeaderElectionRecord{
 		LeaderTransitions: le.observedRecord.LeaderTransitions,
+		LeaseDurationSeconds: le.observedRecord.LeaseDurationSeconds,
```

Comment:
There should be a UT to validate, and the error you reported should have denoted `Failed to release lock:` but I had not seen it.  

It's just odd why you would add a duration when you are releasing the lock...